nucypher/cli/commands/alice.py

```diff
@@ -392,13 +408,13 @@ def grant(general_config,
         'bob_encrypting_key': bob_encrypting_key,
         'bob_verifying_key': bob_verifying_key,
         'label': label,
-        'm': m,
-        'n': n,
+        'm': config_options.m,
+        'n': config_options.n,
         'expiration': expiration,
     }
 
     if not ALICE.federated_only:
-        grant_request.update({'value': value, 'rate': rate})
+        grant_request.update({'rate': config_options.rate})
```

Comment:
I thought we allowed either `value` or `rate`? Should we check which one is provided and then add one to the request?

Reply:
@mswilkison had the same comment. It seems that @KPrasch's intention is to disable value altogether, I guess for UX reasons. I must say I agree with this, since value is not very easy to compute for an end-user.